src/main/com/yetanalytics/datasim/input.clj

```diff
@@ -56,6 +65,35 @@
 (s/def ::parameters
   ::params/parameters)
 
+(defn- validate-profiles
+  [profile-coll]
+  (if (vector? profile-coll)
+    (let [prof-errs (pan/validate-profile-coll profile-coll
+                                               :syntax? true
+                                               :pattern-rels? true
```

Comment:
just pattern, not template?

Reply:
`:template-rels?` is about relationships between Templates and Concepts. `:pattern-rels?` is about relations between Patterns and Templates, as well as between different Patterns.